src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_validators.py

```diff
@@ -0,0 +1,19 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+import re
+
+from azure.cli.core._util import CLIError
+
+def validate_registry_name(namespace):
+    if namespace.registry_name:
+        registry_name = namespace.registry_name
+        if len(registry_name) < 5 or len(registry_name) > 60:
```

Comment:
I agree that for better consistency the validation should only be done on the server side. This simple validation here is meant to provide a quick response to frequent errors in registry names (such as length, dash-is-not-allowed, etc.). #ByDesign @tjprescott